specify/specify7
# Prevent user from saving CO without current determination

Fixes #4819

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Go to Data Entry->Collection Object
- Add multiple determinations
- Play around with deleting, unchecking and adding a current determination
- [ ] Verify save is blocked when no current determinations exist
- [ ] Verify save is enabled when a current determination is added
